src/tables.ts

```diff
@@ -555,52 +555,6 @@ export class Tables {
     return clefs[clef];
   }
 
-  /** Use the provided key to look up a FontInfo in CommonMetrics. **/
-  static lookupMetricFontInfo(key: string): Required<FontInfo> {
```

Comment:
Do you also plan to delete the big `CommonMetrics` from tables.ts?

Reply:
That was the aim, and I forgot :). Done